[2024-10-23T00:30:40.920+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-23T00:30:40.980+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: aula_4_1_get_stocks_dag.AAPL scheduled__2022-01-20T00:00:00+00:00 [queued]>
[2024-10-23T00:30:41.008+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: aula_4_1_get_stocks_dag.AAPL scheduled__2022-01-20T00:00:00+00:00 [queued]>
[2024-10-23T00:30:41.009+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-23T00:30:41.047+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): AAPL> on 2022-01-20 00:00:00+00:00
[2024-10-23T00:30:41.065+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=208) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-23T00:30:41.067+0000] {standard_task_runner.py:72} INFO - Started process 272 to run task
[2024-10-23T00:30:41.072+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'aula_4_1_get_stocks_dag', 'AAPL', 'scheduled__2022-01-20T00:00:00+00:00', '--job-id', '316', '--raw', '--subdir', 'DAGS_FOLDER/aula_4_1_get_stocks.py', '--cfg-path', '/tmp/tmpu9swuvvm']
[2024-10-23T00:30:41.075+0000] {standard_task_runner.py:105} INFO - Job 316: Subtask AAPL
[2024-10-23T00:30:41.181+0000] {task_command.py:467} INFO - Running <TaskInstance: aula_4_1_get_stocks_dag.AAPL scheduled__2022-01-20T00:00:00+00:00 [running]> on host 375c06cd6615
[2024-10-23T00:30:41.377+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='aula_4_1_get_stocks_dag' AIRFLOW_CTX_TASK_ID='AAPL' AIRFLOW_CTX_EXECUTION_DATE='2022-01-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-01-20T00:00:00+00:00'
[2024-10-23T00:30:41.379+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-23T00:30:41.390+0000] {cache.py:130} INFO - Failed to create TzCache, reason: Error creating TzCache folder: '/home/***/.cache/py-yfinance' reason: [Errno 13] Permission denied: '/home/***/.cache/py-yfinance'. TzCache will not be used. Tip: You can direct cache to use a different location with 'set_tz_cache_location(mylocation)'
[2024-10-23T00:30:41.391+0000] {cache.py:336} INFO - Failed to create CookieCache, reason: Error creating CookieCache folder: '/home/***/.cache/py-yfinance' reason: [Errno 13] Permission denied: '/home/***/.cache/py-yfinance'. CookieCache will not be used. Tip: You can direct cache to use a different location with 'set_tz_cache_location(mylocation)'
[2024-10-23T00:30:44.851+0000] {history.py:245} ERROR - $AAPL: possibly delisted; no price data found  (1h 2022-01-19 -> 2022-01-20) (Yahoo error = "1h data not available for startTime=1642568400 and endTime=1642654800. The requested range must be within the last 730 days.")
[2024-10-23T00:30:44.899+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-23T00:30:45.119+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-23T00:30:45.120+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=aula_4_1_get_stocks_dag, task_id=AAPL, run_id=scheduled__2022-01-20T00:00:00+00:00, execution_date=20220120T000000, start_date=20241023T003040, end_date=20241023T003045
[2024-10-23T00:30:45.187+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-23T00:30:45.215+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
